pkgs/development/python-modules/xdg/default.nix

```
@@ -14,7 +14,7 @@ buildPythonPackage rec {
     owner = "srstevenson";
     repo = pname;
     rev = version;
-    sha256 = "13kgnbwam6wmdbig0m98vmyjcqrp0j62nmfknb6prr33ns2nxbs2";
+    sha256 = "sha256-WMY9Hs9SbErTp8hVBoCWTz4dLQhuzqTRpXFEyE7+woo=";
   };
 
   nativeBuildInputs = [ poetry ];
```

Upstream switched to [`poetry-core`](https://github.com/srstevenson/xdg/releases/tag/5.0.1).

```suggestion
  nativeBuildInputs = [ poetry-core ];
```

Also, requires an update of [line 3](https://github.com/NixOS/nixpkgs/pull/109074/files#diff-d8f40ff218187282ffe449456775e5eb6145275e2b1dfa99ef5b377dce112334L3).

---

thanks for the catch